source/OptionsState.hx

```diff
@@ -98,6 +103,31 @@ class OptionsState extends MusicBeatState
 
 				case 'Preferences':
 					openSubState(new PreferencesSubstate());
+					
+					
+				case "Deuteranopia":
+					OptionsState.Deuteranopiabool = true;
+					OptionsState.Protanopiabool = false;
+					OptionsState.Tritanopiabool = false;
+					trace("Deuteranopia");
```

Comment:
These should just be a single enum-type variable.

Reply:
> These should just be a single enum-type variable.

alr